cpp/arcticdb/version/version_map_batch_methods.hpp

```diff
@@ -117,6 +120,130 @@ inline std::shared_ptr<std::unordered_map<std::pair<StreamId, VersionId>, AtomKe
     return output;
 }
 
+struct StreamVersionData {
```

Comment:
I'd split the whole thing into a cpp and header and put the implementation in the cpp.

Reply:
@willdealtry what is exactly the reason why version_map/version_map_batch methods don't have any .cpp file currently?

Reply:
Okay, having spoken to Gem these changes are also included in #415. This is an additional commit on top of that.